ops/pebble.py

```diff
@@ -775,6 +873,52 @@ def __repr__(self):
                 ).format(self=self)
 
 
+class CheckInfo:
+    """Check status information."""
+
+    def __init__(
+        self,
+        name: str,
+        level: str,
+        status: str,
+        failures: int = 0,
+        threshold: int = 0,
+    ):
+        self.name = name
+        self.level = level
+        self.status = status
+        self.failures = failures
+        self.threshold = threshold
+
+    @classmethod
+    def from_dict(cls, d: typing.Dict) -> 'CheckInfo':
```

Comment:
Same here. Or is this 'legacy code style'?

Reply:
See above.